__author__ = 'stephen'


def error_exceptions():
    list = []





#     get_numbers = raw_input('give me a list if numbers, like 2,5,3,4:   ')
#     return range(get_numbers)
# try:
#     number_range = get_numbers()
# except ValueError:
#     print 'the numbers you entered are not in alist.'






























    # user_name = raw_input("Please enter your name:  ")
    # if user_name == ('Stephen'):
    #     return 'that is the right name'
    #
    # elif user_name == ('John Doe'):
    #     return 'You are not an Authorized user here.'
    #

# # try:
# #     user_name = entered_name()
# # except NameError:
#     print 'the name is not correct'